openmrs/apps/registration/app.json

```diff
@@ -109,7 +109,12 @@
                     "label": "REGISTRATION_SEARCH_ADDRESS_LABEL_KEY",
                     "placeholder": "REGISTRATION_SEARCH_ADDRESS_PLACEHOLDER_KEY",
                     "field": "address1"
-                }
+                }, 
+                "customAttributes": {
+                        "label": "REGISTRATION_SEARCH_PHONE_NUMBER",
```

Comment:
@prashantsable1991 Please fix indenting on this file :)

Reply:
Added indentations please review once.

Reply:
> @prashantsable1991 Please fix indenting on this file :)

@lauravignoli As per your suggestion apply indentation , I applied indentation on whole file. I think it is good practices to do indentation on whole file cause in the end we should look to whole file. 
Going ahead we should follow standard indentation cause every individual developers 
thinking is different for indentation.